plse/data.py

```diff
@@ -137,13 +137,26 @@ def one_hot_encode_with_overflow(n, n_max):
 
 
 class DataGenerator(Sequence):
-    def __init__(self, x, y, n_min, n_max, batch_size=2**13, shuffle=True):
+    def __init__(self, x, y, batch_size=2**13, shuffle=True, augment_data=True, n_min=-10, n_max=10):
+        """
+        DataGenerator constructor.
+
+        Parameters:
+            x (np.ndarray): The input data for the model, 2d waveform data.
+            y (np.ndarray): The true output data for the model, encoded_npe data or 2D time data.
+            batch_size(int): The batch size.
+            shuffle (bool): Whether to shuffle at the end of each epoch.
+            augment_data (bool): Whether to shift waveform data to left and right, as augmentation.
+            n_min (int): Maximum number of bins to shift to the left when augmenting waveform data.
```

Comment:
It may be useful to refactor n_min, n_max to n_left, n_right and change their type to unsigned ints to avoid confusion and better match the definitions provided

Reply:
Maybe n_max_left and n_max_right? So it doesn't sound like that's the exact # of bins left or right to move

Reply:
Proposed solution at line 155:
```
assert max_shift_right>=0, "Maximum shift to the right should not be negative"
self.max_shift_left = -1*abs(max_shift_left)
self.max_shift_right = abs(max_shift_right)
```

Reply:
Great, thanks!